helm/hpcc/templates/roxie.yaml

```diff
@@ -161,6 +163,7 @@ spec:
         roxie-server: {{ $servername | quote }}
 {{- end }}
     spec:
+      serviceAccountName: "hpcc-default"
```

Comment:
trivial: should we be quoting all strings for consistency? e.g. here vs below in service-account.yaml name:

Reply:
Not sure I care enough to go through and change...

Reply:
fair enough.